frontend/src/scenes/themes/CustomCssScene.tsx

```diff
@@ -78,6 +78,223 @@ body[theme=light] {
     --secondary-3000-button-border-hover: #d40b76;
 }`
 
+const LUMON_THEME = /* === Import Input Sans Font from Google Fonts === */
```

Comment:
You're missing backticks here at the start and end of the string

```suggestion
const LUMON_THEME = `
/* === Import Input Sans Font from Google Fonts === */
```